src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_common.py

```diff
@@ -124,16 +125,3 @@ def database_delete_func(client, resource_group_name=None, server_name=None, dat
         except Exception as ex:  # pylint: disable=broad-except
             logger.error(ex)
     return result
-
-
-def user_confirmation(message, yes=False):
-    if yes:
-        return True
-    from knack.prompting import prompt_y_n, NoTTYException
-    try:
-        if not prompt_y_n(message):
-            raise CLIError('Operation cancelled.')
-        return True
-    except NoTTYException:
-        raise CLIError(
-            'Unable to prompt for confirmation as no tty available. Use --yes.')
```

Comment:
Check where this function is called. It could be refactored without checking the return value.